tsfresh/__init__.py

```diff
@@ -18,6 +18,18 @@
     __version__ = 'unknown'
 
 
+# Set default logging handler to avoid "No handler found" warnings.
+import logging
+try:  # Python 2.7+
+    from logging import NullHandler
```

Comment:
I was thinking about setting a void logger (something that just writes all logging messages to /dev/null) somewhere in the top level in extract_features if no logging backend is specified (when setting the void logger one can print a short warning "hey we found no logger, going to drop all messages from here on"). However, my knowledge of the logging package is limited.